scripts/west_commands/completion/west-completion.zsh

```diff
@@ -102,9 +102,12 @@ _get_west_projs() {
 }
 
 _get_west_boards() {
-  _west_boards="$(__west_x boards --format={identifiers})\n$(__west_x boards --format={name})"
-  _west_boards=${_west_boards//$'\n'/\ }
-  _west_boards=${_west_boards//,/\ }
+  _west_boards=( $(__west_x boards --format='{name}|{identifiers}') )
+  for i in {1..${#_west_boards[@]}}; do
+    local name="${_west_boards[$i]%%|*}"
```

Comment:
zsh autocomplete does not work: `_get_west_boards:5: _west_boards: assignment to invalid subscript range`